Sources/SpeziLLM/Mock/LLMMockSession.swift

```diff
@@ -52,29 +52,19 @@ public final class LLMMockSession: LLMSession, @unchecked Sendable {
                 return
             }
             
-            /// Generate mock messages
             await MainActor.run {
                 self.state = .generating
             }
-            await injectAndYield("Mock ", on: continuation)
-            
-            try? await Task.sleep(for: .milliseconds(500))
-            guard await !checkCancellation(on: continuation) else {
-                return
-            }
-            await injectAndYield("Message ", on: continuation)
-            
-            try? await Task.sleep(for: .milliseconds(500))
-            guard await !checkCancellation(on: continuation) else {
-                return
-            }
-            await injectAndYield("from ", on: continuation)
             
-            try? await Task.sleep(for: .milliseconds(500))
-            guard await !checkCancellation(on: continuation) else {
-                return
+            /// Generate mock messages
+            let tokens = ["Mock ", "Message ", "from ", "SpeziLLM!"]
+            for token in tokens {
+                try? await Task.sleep(for: .milliseconds(500))
+                guard await !checkCancellation(on: continuation) else {
```

Comment:
nit: I usually try to void double negations since they make the code harder to read. Instead we can use `if`.

Reply:
We used this pattern throughout the code base, might make sense to simplify it. Definitely no must tho @LeonNissen

Reply:
Yeah exactly @philippzagar, but I see your point @jdisho. Maybe a todo for later?